src/opencensus.erl

```diff
@@ -85,8 +84,11 @@ start_span(Name, TraceId, ParentId) when is_integer(TraceId)
 -spec finish_span(maybe(span())) -> maybe(span()).
 finish_span(undefined) ->
     undefined;
-finish_span(Span) ->
-    Span#span{end_time = timestamp()}.
+finish_span(Span=#span{start_time=StartTime}) ->
+    EndTime = wts:timestamp(),
+    Span#span{start_time = StartTime,
```

Comment:
Does it make sense to mutate start time? Not sure if it's optimized away but it seems redundant.

Reply:
Dammit, hehe, no, that is a mistake, one sec.